File: C_Workspace/C_CheckList/Basics/5.c
//Program to find largest number among three numbers.
#include <stdio.h>

int main() 
{
    float a,b,c;
    printf("Enter the number:");
    scanf("%f",&a);
    printf("Enter the number:");
    scanf("%f",&b);
    printf("Enter the number:");
    scanf("%f",&c);    
    if(a>b && a>c)
    {
    printf("%.2f is greater than %.2f %.2f",a,b,c);
    }
    else if(b>a && b>c)
    {
    printf("%.2f is greater than %.2f %.2f",b,a,c);
    }
    else
    {
    printf("%.2f is greater than %.2f %.2f",c,a,b);
    }
}

File: Python_Workspace/Python_Tasks/15_06/5.py
#Greatest of the Three numbers
x=input("Enter 1st number:")
y=input("Enter 2nd number:")
z=input("Enter 3rd number:")
if(x>y and x>z):
  print(x,"is greater")
elif(x<y and y>z):
  print(y,"is greater")
else:
  print(z,"is greater")
